cmd/osbuild-composer/composer.go

```diff
@@ -402,7 +402,7 @@ func (c *Composer) Start() error {
 func (c *Composer) ensureStateDirectory(name string, perm os.FileMode) (string, error) {
 	d := path.Join(c.stateDir, name)
 
-	err := os.Mkdir(d, perm)
+	err := os.MkdirAll(d, perm)
```

Comment:
Can probably drop this, I was trying to run composer from source locally but I gave up.

Reply:
Dropping.